distribution/lib/Standard/Database/0.0.0-dev/src/Data/Column.enso

```diff
@@ -174,29 +177,6 @@ type Column
     make_unary_op : Text -> Text -> (SQL_Type | Nothing) -> Column
     make_unary_op self op_kind new_type=Nothing = self.make_op op_kind [] new_type
 
-    ## UNSTABLE
-
-       Efficiently joins two tables based on either the index or a key column.
-
-       Arguments:
-       - other: the table being the right operand of this join operation.
-       - on: the column(s) or expression(s) of `self` that should be used as
-         the join key. If this argument is not provided, the index of `self`
-         will be used.
-       - drop_unmatched: whether the rows of `self` without corresponding
-         matches in `other` should be dropped from the result.
-       - left_suffix: a suffix that should be added to the columns of `self`
-         when there's a name conflict with a column of `other`.
-       - right_suffix: a suffix that should be added to the columns of `other`
-         when there's a name conflict with a column of `self`.
-
-       The resulting table contains rows of `self` extended with rows of
-       `other` with matching indexes. If the index in `other` is not unique,
-       the corresponding rows of `self` will be duplicated in the result.
-    join : Table | Column -> Nothing | Text | Column | Vector (Text | Column) -> Boolean -> Text -> Text -> Table
-    join self other on=Nothing drop_unmatched=False left_suffix='_left' right_suffix='_right' =
-        self.to_table.join other on drop_unmatched left_suffix right_suffix
```

Comment:
This is a leftover from the old table joins API, it was not working after new Joins were introduced, but it was never tested so it rotted here.